### ISSUE TITLE
Error when using bundle-stats v4.14.0

### ISSUE THREAD

**Issue by BPScott** [relative-ci/bundle-stats#4636]
The changes in [bundle-stats v4.14.0](https://github.com/relative-ci/bundle-stats/releases/tag/v4.14.0) that let you identify what chunks a module belongs to looks super useful.

However the stats file produced by rollup-plugin-webpack-stats `v1.0.2` causes a runtime error when clicking a module in the `modules` tab.

```
bundle-stats.html:2 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'size')
    at Cs (bundle-stats.html:2:311443)
```

The UI seems to be unable to recover from this error, and I am no longer able to click into other tabs.

I suspect `size`, and probably other pieces of information that were previously not required by `bundle-stats`, are now required, and as this package doesn't aim to produce a complete stats file, only including what bundle-stats needs, it hasn't been updated yet to include those new required data keys.

### Reproduction steps

- Run a Vite build that has been configured to use rollup-plugin-webpack-stats `v1.0.2`. (the repo I'm using is private so I don't want to post the generated stats file publicly, but I suspect any vite build would work).
- Pass the generated stats file into bundle-stats v4.14.0
- Open up the html report, click the modules tab, then click on a module in the table. See the error above appear in the console.

**Comment by vio** [comment]
Was able to replicate the issue for cases when the report has no baseline, that's a case we are not testing automatically.

If you get the chance, can you please try `v4.14.1-beta.0` ?

**Comment by vio** [comment]
@BPScott thank you for checking! The fix is now available on [`v4.14.1`](https://github.com/relative-ci/bundle-stats/releases/tag/v4.14.1).

> As an aside/polish idea: It feels a little odd to show size change information when looking at a single build. I'd be tempted to consider removing the +2.91KiB / +100% red diff text when there is no baseline.

Indeed, we do not know if the baseline value is missing or just empty, will handle it here: https://github.com/relative-ci/bundle-stats/issues/4639. Thank you for the feedback!
